src/components/carmel-gearvr-controls.js

```diff
@@ -0,0 +1,200 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var getGamepadsByPrefix = require('../utils/tracked-controls').getGamepadsByPrefix;
+var THREE = require('../lib/three');
+
+var GAMEPAD_ID_PREFIX = 'Gear VR Touchpad';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('carmel-gearvr-controls', {
+  // since aframe-teleport-controls looks for tracked-controls, add one
+  dependencies: ['tracked-controls'],
```

Comment:
@donmccurdy i agree this could be more generic like touchpad-controls if the behavior were consistent across browsers (not just Carmel) and controllers (not just Gear VR Touchpad), but it is not clear that is the case.  happy to rename if that helps.